SS14.Launcher/Helpers.cs

```diff
@@ -171,4 +171,11 @@ public static void ChmodPlusX(string path)
             FileAccessPermissions.UserExecute | FileAccessPermissions.GroupExecute |
             FileAccessPermissions.OtherExecute;
     }
+    public static unsafe int MessageBoxHelper(string text, string caption, uint type)
+    {
+        fixed (char* pText = text)
+        fixed (char* pCaption = caption)
+
+        return Windows.MessageBoxW(HWND.NULL, (ushort*)pText, (ushort*)pCaption, type);
+    }
```

Comment:
Absolutely do not:
1. Use a braceless block for a `fixed()` block.
2. Un-indent a braceless block.